beetsplug/acousticbrainz.py

```diff
@@ -151,7 +176,7 @@ def _get_data(self, mbid):
                 return {}
 
             if res.status_code == 404:
-                self._log.info(u'recording ID {} not found', mbid)
+                self._log.info(u'recording ID \'{}\' not found', mbid)
```

Comment:
Any particular reason to add these single quotes in these log statements? For the sake of the cleanliness of this PR, it would be awesome to leave that to a later change if you don't mind—and keep this change focused on just the fix you have here.

Reply:
I did this while debugging ... the output was easier to read for me this way.

Example:
```
acousticbrainz: attribute 'gender' of 'M83 -  - Ludivine' set to 'male'
```

But, anyway, you are correct. I can revert it if you like.

Reply:
It's probably best to revert this if not just because it adds unrelated changes to this PR.

Reply:
Reverted them.